src/plays/tic-tac-toe/TicTacToe.jsx

```diff
@@ -0,0 +1,29 @@
+import { getPlayById } from "meta/play-meta-util";
+
+import PlayHeader from "common/playlists/PlayHeader";
+
+// importing Game component
```

Comment:
Do we really need this comment? I think import is saying what are you doing here, so comment is useless